docs/content/release_notes/whats_new.md

```diff
@@ -4,11 +4,28 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
 
+## New dynamic volume group support
+
+The new volume group support allows dynamic management of the content of the groups. This feature is used by policy-based replication, introduced IBM Spectrum Virtualize 8.5.2 release.
+As opposed to the volume group feature present within the CSI driver which stays static on the StorageClass, the new volume group feature is dynamic. 
+
+With this new volume group support, the PersistentVolumeClaim (PVC) label can be updated at any time to change the PVC from belonging from one volume group to another, once the volume groups are defined on the storage.
+
+The advantage of using volume groups is that actions, like replication, can be done simultaneously across all volumes in a volume group.
+
+For more information about volume groups and policy-based replication, see the following sections within your Spectrum Virtualize product documentation [IBM Documentation](https://www.ibm.com/docs).
+
+- **Product overview** > **Technical overview** > **Volume groups**
+- **What's new** > **Getting started with policy-based replication**
+
+**Important:** Volume groups need to be used together with policy-based replication. For more information, see [Using the CSI driver with policy-based replication](../using/using_policy_based_replication.md).
 
 ## New support for policy-based replication
 
 This version adds support for policy-based replication that was introduced in IBM Spectrum Virtualize 8.5.2 release. Policy-based replication provides a simplified configuration and management of asynchronous replication between two system. To see if your specific product is supported and for more information, see **What's new** > **Getting started with policy-based replication** within your Spectrum Virtualize product documentation on [IBM Documentation](https://www.ibm.com/docs).
 
+**Important:** Volume groups is an integral part of policy-based replication and must be used together. For more information, see [Using the CSI driver with policy-based replication](../using/using_policy_based_replication.md).
```

Comment:
the users can still use volume groups without policy-based replication.
it is that he can't use policy-based replication without the volume groups.

Reply:
@ArbelNathan  please confirm. it was my understanding that it was both ways. thanks

Reply:
updated accordingly in webex with Arbel